zerver/lib/actions.py

```diff
@@ -5454,6 +5454,11 @@ def send_change_stream_description_notification(
     sender = get_system_bot(settings.NOTIFICATION_BOT, acting_user.realm_id)
     user_mention = silent_mention_syntax_for_user(acting_user)
 
+    if len(new_description) == 0:
+        new_description = "*No description*"
+    if len(old_description) == 0:
+        old_description = "*No description*"
```

Comment:
I think `new_description.strip() == ""` would be a better test here; or maybe just `== ""` if we strip the description when checking it, which is very possible.

Reply:
Update the PR.
Thanks :)